tests/pages/firefox/developer.py

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from .base import FirefoxBasePage
+from ..page import PageRegion
+
+
+class DeveloperPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/developer'
+    _primary_download_locator = (By.CSS_SELECTOR, '.intro .download-button')
+    _secondary_download_locator = (By.CSS_SELECTOR, '.dev-footer-download .download-button')
+
+    @property
+    def is_primary_download_button_displayed(self):
+        return self.selenium.find_element(*self._primary_download_locator).is_displayed()
```

Comment:
Just going by example of other tests here, but I think it's a good suggestion. I'll add a method for this to `page.py`